pkgs/top-level/python-packages.nix

```diff
@@ -9141,6 +9141,8 @@ self: super: with self; {
 
   nltk = callPackage ../development/python-modules/nltk { };
 
+  textblob = callPackage ../development/python-modules/textblob { };
```

Comment:
This file should be sorted alphabetically.

Reply:
There are some package not in order. I have added textblob in right place.